test/TestConfig/extraSettings

```diff
@@ -48,28 +48,11 @@ ifneq (,$(findstring cmprssptrs,$(SPEC)))
 VM_SUBDIR=compressedrefs
 endif
 
-ifneq (,$(findstring le,$(SPEC)))
-	ARCH_DIR=ppc64le
-else ifneq (,$(findstring ppc-64,$(SPEC)))
-	ARCH_DIR=ppc64
-else ifneq (,$(findstring ppc,$(SPEC)))
-	ARCH_DIR=ppc
-else ifneq (,$(findstring arm,$(SPEC)))
-	ARCH_DIR=arm
-else ifneq (,$(findstring 390-64,$(SPEC)))
-	ARCH_DIR=s390x
-else ifneq (,$(findstring 390,$(SPEC)))
-	ARCH_DIR=s390
-else ifneq (,$(findstring linux_x86-64,$(SPEC)))
-	ARCH_DIR=amd64
-else ifneq (,$(findstring linux_x86,$(SPEC)))
-	ARCH_DIR=i386
-endif
-
 ifneq (,$(findstring win,$(SPEC)))
 	JAVA_SHARED_LIBRARIES_DIR:=$(JAVA_BIN)$(D)$(VM_SUBDIR)
```

Comment:
Is there a reason to keep the windows special case?

Reply:
I don't think windows uses LIBPATH at all.  I was planning on doing a search and testing to further cleanup this file before removing the WIP.